scripts/sync.js

```diff
@@ -45,7 +46,7 @@ projectNames.forEach((project) => {
 })
 
 if (updatedVersion || program.init || program.run_sync) {
-  util.gclientSync({verbose: config.gClientVerbose})
+  util.gclientSync({verbose: config.gClientVerbose, cache_dir: config.gClientCacheDir})
```

Comment:
Options passed to util.gclientSync changed a bit with #300. We need to get that merged, but I'd rebase off that if you can.

Reply:
Merged that pr, so should be able to rebase off master now

Reply:
Fixed by altering the code inside gclientSync to just pull form the `config` object.